website/docs/modules/create.html.markdown

```diff
@@ -118,8 +118,14 @@ standard structure.
 
 * **README**. The root module and any nested modules should have README
   files. This file should be named `README` or `README.md`. The latter will
-  be treated as markdown. The README doesn't need to document inputs or
-  outputs of the module because tooling will automatically generate this.
+  be treated as markdown. There should be a description of the module and 
+  what it should be used for, as well as a basic usage example. Consider
```

Comment:
I think I mentioned before but did we decide on encouraging usage examples? It seems this duplicates the generated usage block at the top right?

Reply:
There should be a description of the module and what it should be used for. If you want to include an example for how this module can be used in combination with other resources, put it in an [examples directory like this](https://github.com/hashicorp/terraform-aws-consul/tree/master/examples)

Reply:
@KFishner we already call out the examples in a section below this:

 "Examples. Examples of using the module should exist under the examples/ subdirectory. Each example may have a README to explain the goal and usage of the example." 

Should we mention the examples in the README section as well?

Reply:
yeah I think it's a good idea to call it out in the README